### PR TITLE
content fixes

### PR DESCRIPTION
**Pull request by AlexMurphy**

### CODE REVIEW

File: acceptance_tests/features/step_definitions/DSP-96_.steps.rb [hunk 1]
```diff
@@ -46,7 +46,7 @@
   fill_in('date-of-birth-year', :with => '1988')
   fill_in('nationality', :with => 'Chinese')
   click_button('Continue')
-  fill_in('email', :with => 'alex.murphy@uk.sogeti.com')
+  fill_in('email', :with => 'alexandermurphy1988@yahoo.co.uk')
```
Comment (easternbloc) [inline acceptance_tests/features/step_definitions/DSP-96_.steps.rb]: perhaps we should have a test email?

Comment (JoeChapman) [inline acceptance_tests/features/step_definitions/DSP-96_.steps.rb]: lol +1

File: acceptance_tests/features/step_definitions/DSP-92_.steps.rb [hunk 2]
```diff
@@ -32,5 +32,5 @@
 
 Then(/^I am on Step Three of the Lost\/Stolen form$/) do                                                
   page.should have_content('Step 3 of 5')                                               
-  page.should have_content('What are your personal details?')
+  page.should have_content('What are your personal details')
```
Comment (JoeChapman) [inline acceptance_tests/features/step_definitions/DSP-92_.steps.rb]: This probably should have a question mark. Have I removed the question mark from the code?

**Comment by JoeChapman** [comment]
Dependent on https://github.com/UKHomeOffice/brp_app/pull/113

File: acceptance_tests/features/step_definitions/DSP-96_.steps.rb (hunk 1 above)
Comment (easternbloc) [inline acceptance_tests/features/step_definitions/DSP-96_.steps.rb]: @AlexMurphy what do you want to do here?

**Pull request merged**